src/coreclr/tools/Common/JitInterface/CorInfoBase.cs

```diff
@@ -2776,3 +2777,4 @@ private static IntPtr GetUnmanagedCallbacks()
         }
     }
 }
+
```

Comment:
Looks like a lot of failures caused by this newline.

Reply:
Seems to be an enforced code style analyzer, possibly specific to the `JitInterface` cs files, since it isn't one we enforce in corelib or more broadly to my knowledge.

Reply:
I have fixed that. I do still see some superpmi failures. I do not expect any diffs in code since we are not really modifying codegen. Should I look into these or are these known issues? This looks like a infra failure from what I understand but do let me know if I have misunderstood the below logs(Esp following - **_No Azure Storage MCH files to download from eb8352bd-0a13-4b5b-badb-58f9ecc40c44/linux/arm64/_**)


```
Generating script.
2022-09-13T23:15:27.2899347Z ========================== Starting Command Output ===========================
2022-09-13T23:15:27.3168856Z ##[command]"C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe" -NoLogo -NoProfile -NonInteractive -ExecutionPolicy Unrestricted -Command ". 'D:\a\_work\_temp\35ae31c7-4cfd-466d-bdda-8e97e6756db0.ps1'"
2022-09-13T23:15:30.6143525Z Setting NUGET enhanced retry environment variables
2022-09-13T23:15:30.6657992Z Setting NUGET enhanced retry environment variables
2022-09-13T23:15:31.0448738Z D:\a\_work\1\s\.dotnet\sdk\7.0.100-preview.7.22377.5\MSBuild.dll /nologo -maxcpucount /m /maxcpucount -verbosity:m /v:minimal /bl:D:\a\_work\1\s/artifacts/log/SendToHelix.binlog /clp:Summary /logger:D:\a\_work\1\s\.packages\microsoft.dotnet.arcade.sdk\8.0.0-beta.22455.1\tools\net7.0\Microsoft.DotNet.ArcadeLogging.dll /nr:False /p:ContinuousIntegrationBuild=True /p:TreatWarningsAsErrors=false /p:TargetArchitecture=x64 /p:TargetOS=windows /p:TargetOSSubgroup= /p:Configuration=checked D:\a\_work\1\s/src/coreclr/scripts/superpmi-replay.proj
2022-09-13T23:15:34.2433810Z   Sending Job to Windows.10.Amd64.Open...
2022-09-13T23:15:36.6819223Z   Sent Helix Job; see work items at https://helix.dot.net/api/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems?api-version=2019-06-17
2022-09-13T23:15:37.7129910Z   Waiting for completion of job 6935e4d0-ad31-4de8-843e-519be36c722a on Windows.10.Amd64.Open
2022-09-13T23:37:01.6476418Z   Job 6935e4d0-ad31-4de8-843e-519be36c722a on Windows.10.Amd64.Open is completed with 11 finished work items.
2022-09-13T23:37:07.7705162Z   Downloading result files for job 6935e4d0-ad31-4de8-843e-519be36c722a
2022-09-13T23:37:07.8817262Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Work item linux-arm64-1 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed. [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.8820413Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/linux-arm64-1/console [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.8947072Z ##[error].packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : (NETCORE_ENGINEERING_TELEMETRY=Test) Work item linux-arm64-1 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed.
Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/linux-arm64-1/console
2022-09-13T23:37:07.8967391Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Work item linux-arm64-2 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed. [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.8972720Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/linux-arm64-2/console [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.8978502Z ##[error].packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : (NETCORE_ENGINEERING_TELEMETRY=Test) Work item linux-arm64-2 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed.
Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/linux-arm64-2/console
2022-09-13T23:37:07.8982267Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Work item osx-arm64-1 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed. [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.8984930Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/osx-arm64-1/console [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.8995906Z ##[error].packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : (NETCORE_ENGINEERING_TELEMETRY=Test) Work item osx-arm64-1 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed.
Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/osx-arm64-1/console
2022-09-13T23:37:07.8998310Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Work item osx-arm64-2 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed. [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9000977Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/osx-arm64-2/console [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9010591Z ##[error].packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : (NETCORE_ENGINEERING_TELEMETRY=Test) Work item osx-arm64-2 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed.
Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/osx-arm64-2/console
2022-09-13T23:37:07.9014864Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Work item unix-x64-1 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed. [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9017581Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/unix-x64-1/console [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9022925Z ##[error].packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : (NETCORE_ENGINEERING_TELEMETRY=Test) Work item unix-x64-1 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed.
Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/unix-x64-1/console
2022-09-13T23:37:07.9025184Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Work item unix-x64-2 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed. [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9027264Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/unix-x64-2/console [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9032253Z ##[error].packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : (NETCORE_ENGINEERING_TELEMETRY=Test) Work item unix-x64-2 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed.
Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/unix-x64-2/console
2022-09-13T23:37:07.9034482Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Work item win-arm64-1 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed. [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9035897Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/win-arm64-1/console [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9040554Z ##[error].packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : (NETCORE_ENGINEERING_TELEMETRY=Test) Work item win-arm64-1 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed.
Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/win-arm64-1/console
2022-09-13T23:37:07.9042639Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Work item win-arm64-2 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed. [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9043991Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/win-arm64-2/console [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9050019Z ##[error].packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : (NETCORE_ENGINEERING_TELEMETRY=Test) Work item win-arm64-2 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed.
Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/win-arm64-2/console
2022-09-13T23:37:07.9052673Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Work item win-x64-1 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed. [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9054225Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/win-x64-1/console [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9059145Z ##[error].packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : (NETCORE_ENGINEERING_TELEMETRY=Test) Work item win-x64-1 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed.
Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/win-x64-1/console
2022-09-13T23:37:07.9061579Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Work item win-x64-2 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed. [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9063244Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/win-x64-2/console [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9067693Z ##[error].packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : (NETCORE_ENGINEERING_TELEMETRY=Test) Work item win-x64-2 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed.
Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/win-x64-2/console
2022-09-13T23:37:07.9069142Z 
2022-09-13T23:37:07.9069434Z Build FAILED.
2022-09-13T23:37:07.9069722Z 
2022-09-13T23:37:07.9070536Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Work item linux-arm64-1 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed. [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9071891Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/linux-arm64-1/console [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9073394Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Work item linux-arm64-2 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed. [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9074761Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/linux-arm64-2/console [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9076273Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Work item osx-arm64-1 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed. [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9077833Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/osx-arm64-1/console [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9079305Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Work item osx-arm64-2 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed. [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9080653Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/osx-arm64-2/console [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9081985Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Work item unix-x64-1 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed. [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9083316Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/unix-x64-1/console [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9084730Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Work item unix-x64-2 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed. [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9086095Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/unix-x64-2/console [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9087428Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Work item win-arm64-1 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed. [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9088913Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/win-arm64-1/console [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9090461Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Work item win-arm64-2 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed. [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9091834Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/win-arm64-2/console [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9093351Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Work item win-x64-1 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed. [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9094878Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/win-x64-1/console [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9096338Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Work item win-x64-2 in job 6935e4d0-ad31-4de8-843e-519be36c722a has failed. [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9097798Z D:\a\_work\1\s\.packages\microsoft.dotnet.helix.sdk\8.0.0-beta.22455.1\tools\Microsoft.DotNet.Helix.Sdk.MultiQueue.targets(90,5): error : Failure log: https://helix.dot.net/api/2019-06-17/jobs/6935e4d0-ad31-4de8-843e-519be36c722a/workitems/win-x64-2/console [D:\a\_work\1\s\src\coreclr\scripts\superpmi-replay.proj]
2022-09-13T23:37:07.9098671Z     0 Warning(s)
2022-09-13T23:37:07.9098971Z     10 Error(s)
2022-09-13T23:37:07.9099085Z 
2022-09-13T23:37:07.9099419Z Time Elapsed 00:21:36.76
2022-09-13T23:37:07.9660519Z Build failed with exit code 1. Check errors above.
```


Looking at the logs mentioned there


```
Console log: 'linux-arm64-1' from job 6935e4d0-ad31-4de8-843e-519be36c722a workitem 9b195ed3-9043-4207-9503-c4d5e50d3e2d (windows.10.amd64.open.rt) executed on machine a0026IJ running Windows-10-10.0.14393-SP0

C:\h\w\AA030949\w\9F1D08E5\e>C:\python3\python.exe C:\h\w\AA030949\p\superpmi_replay.py -jit_directory C:\h\w\AA030949\p -arch arm64 -platform Linux -partition 1 -partition_count 2 -log_directory C:\h\w\AA030949\w\9F1D08E5\uploads 
Running superpmi.py download
Running: C:\python3\python.exe C:\h\w\AA030949\p\superpmi.py download --no_progress -target_os Linux -target_arch arm64 -core_root C:\h\w\AA030949\p -spmi_location C:\h\w\AA030949\p\artifacts\spmi -log_level debug
[23:36:07] ================ Logging to C:\h\w\AA030949\p\artifacts\spmi\superpmi.log

[23:36:07] Using mcs.exe from Core_Root: C:\h\w\AA030949\p\mcs.exe

[23:36:07] Using JIT/EE Version from mcs: eb8352bd-0a13-4b5b-badb-58f9ecc40c44

[23:36:07] SuperPMI download

[23:36:07] ------------------------------------------------------------

[23:36:07] Start time: 23:36:07

**_No Azure Storage MCH files to download from eb8352bd-0a13-4b5b-badb-58f9ecc40c44/linux/arm64/_**

[23:36:08] Finish time: 23:36:08

[23:36:08] Elapsed time: 0:00:00.658097

```

Reply:
> Correction - looks like non exists for this version coz GUID got updated

Right. SuperPMI isn't robust against version GUID changes and so won't give any valid results for this PR.

Reply:
Are you comfortable with leaving it as is or do I run superpmi locally without the GUID change to verify?